CollAction/Views/Manage/Index.cshtml

```diff
@@ -68,22 +68,50 @@
                         </form>
                     </div>
                 </div>
-                @if (!User.IsInRole(Constants.AdminRole))
-                {
-                    <div class="panel panel-default">
-                        <div class="panel-heading">
-                            <dt>Delete Account</dt>
-                        </div>
-                        <div class="panel-body">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <dt>Recurring Donations</dt>
+                    </div>
+                    <div class="panel-body">
+                        @if (Model.DonationSubscriptions.Any())
+                        {
                             <p>
-                                We're sorry if you want to leave... however, if you really want to, you can delete your account here.
+                                You can cancel your recurring donations here.
                             </p>
-                            <form asp-controller="Account" asp-action="Delete" method="post">
-                                <div id="delete-user"></div>
-                            </form>
-                        </div>
+
+                            <div class="panel-body">
+                                @foreach (var subscription in Model.DonationSubscriptions)
+                                {
+                                    <div class="unsubscribe-donation">
+                                        <form asp-controller="Donation" asp-action="CancelSubscription" method="post">
+                                            <input type="hidden" name="subscriptionId" value="@subscription.Id" />
+                                            <input type="submit" value="Cancel donation started at @subscription.Start.Value.ToShortDateString()" />
```

Comment:
This will be a long button.. Maybe put the "Donation started at ..." as text?

Reply:
I just tried some variations, but without "Cancel" it's not really clear that you're stopping your donations (it's not a "details" page). It's a long button, but the layout seems to work. If you have a better variation, go ahead.